tools/dockerise/__init__.py

```diff
@@ -44,113 +44,6 @@ def build_platform(platform):
-
-                        del queue[id]
-                        break
-
-            # Get id and status from the current event
-            id = int(event["id"], 16)
-            status = event["status"]
-
-            # If this is the first time we have seen this id, assign a progress bar if one is available otherwise add
-            # it to the queue
-            if id not in progress and id not in queue:
-                # Check for an available progress bar
-                bar = None
-                for pos in range(num_bars):
-                    if bars[pos]["available"]:
-                        bars[pos]["available"] = False
-                        bars[pos]["id"] = id
-                        bar = bars[pos]["bar"]
-                        break
-
-                if bar is not None:
-                    progress[id] = {"bar": bar}
-                else:
-                    # No available progress bars, add to queue
-                    queue[id] = {"current": 0, "total": 0, "status": status}
-
-            # If we have a value in progressDetail
-            if (
-                "progressDetail" in event
-                and "current" in event["progressDetail"]
-                and "total" in event["progressDetail"]
-            ):
-                current = int(event["progressDetail"]["current"])
-                total = int(event["progressDetail"]["total"])
-            else:
-                current = None
-                total = None
-
-            if id in progress:
-                # Update the status
-                progress[id]["bar"].set_description("{} - {}".format(id, status))
-
-                # Update bar values
-                if current is not None and total is not None:
-                    progress[id]["bar"].total = total
-                    progress[id]["bar"].n = current
-
-                # Complete statuses need to finish off the bar
-                if "complete" in status or "exists" in status:
-                    # Remove from progress
-                    del progress[id]
-
-                    # Free the progress bar
-                    for pos in range(num_bars):
-                        if bars[pos]["id"] == id:
-                            bars[pos]["bar"].set_description("Pending ...")
-                            bars[pos]["bar"].reset()
-                            bars[pos]["id"] = None
-                            bars[pos]["available"] = True
-
-            else:
-                # Find our position in the queue
-                if current is not None and total is not None:
-                    queue[id]["current"] = current
-                    queue[id]["total"] = total
-                queue[id]["status"] = status
-
-        except KeyError:
-            print(event["status"])
-        except ValueError:
-            print(event["status"])
-
-    # Clean up all the bars
-    for k, v in bars.items():
-        v["bar"].reset()
-        v["bar"].close()
-
```

Comment:
Did you intend this?

Reply:
I did not, it doesn't work on my work machine so I delete it